tests/conftest.py

```diff
@@ -153,6 +153,8 @@ def _build_migrations_cache() -> None:
         cluster = storage.get_cluster()
         database = cluster.get_database()
         nodes = [*cluster.get_local_nodes(), *cluster.get_distributed_nodes()]
+        mv_tables = []
+        non_mv_tables = []
```

Comment:
These should be inside the loop otherwise we are creating all tables on all the nodes. It will pass CI but will be an inaccurate representation of the clusters.

Reply:
ah good catch